pkg/prowgen/prowgen_konflux.go

```diff
@@ -412,12 +410,6 @@ func GenerateKonfluxServerlessOperator(ctx context.Context, openshiftRelease Rep
 			GlobalResourcesOutputPath:     resourceOutputPath,
 			PipelinesOutputPath:           fmt.Sprintf("%s/.tekton", r.RepositoryDirectory()),
 			Nudges:                        b.Konflux.Nudges,
```

Comment:
The components also have the `build-nudges-ref`:
https://github.com/openshift-knative/hack/blob/71f8c5394687bc133ab7c0838a6b1364661700af/pkg/konfluxgen/dockerfile-component.template.yaml#L12

Reply:
This stays there

Reply:
The reason being, we might want to use them for some components (like client)

Reply:
:+1: 
/lgtm